bin/common.ml

```diff
@@ -1249,6 +1249,11 @@ let init (builder : Builder.t) =
         }
   in
   Log.info [ Pp.textf "Shared cache: %s" (Config.Toggle.to_string config.cache_enabled) ];
+  Log.info
```

Comment:
Perhaps you should print only if the cache is enabled?

Reply:
We could; but is it worth it? Also, the location of the cache is still valid, even if the cache is disabled _right now_.

Reply:
Sure, I don't think it matters that much. Was just wondering if showing the path to the cache might confuse people that the cache is enabled if they aren't reading carefully.